src/assembler.jl

```diff
@@ -51,3 +51,14 @@ assembled values.
 function end_assemble(a::Assembler)
     return sparse(a.I, a.J, a.V)
 end
+
+"""
+    assemble!(g, ge, edof)
+
+Assembles the element residual `ge` into the global residual vector `g`.
+"""
+@inline function assemble!{T}(g::AbstractVector{T}, ge::AbstractVector{T}, edof::AbstractVector{Int})
+    @inbounds for i in 1:length(ge)
```

Comment:
Do we need to `@assert length(ge) == length(edof) < length(g)` ?
We don't do such a check for the stiffness.